randomizer/RandomizerLocationManager.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+public class RandomizerLocationManager
+{
+	public static void Initialize()
+	{
+		StringReader reader = new StringReader(RandomizerLocationData.All);
```

Comment:
not disposed?

Reply:
Oh hey, I just saw this comment, so I looked up the Microsoft docs for the StringReader class. Here's what I learned:

`This type implements the IDisposable interface, but does not actually have any resources to dispose. This means that disposing it by directly calling Dispose() or by using a language construct such as using (in C#) or Using (in Visual Basic) is not necessary.`

So I guess this specific case is fine as-is, but I'm glad you pointed this out since I haven't seen the `Dispose()` pattern in my limited C# exposure. I'll keep it in mind going forward. Thanks!